src/components/Toolbar.vue

```diff
@@ -6,21 +6,21 @@
     v-spacer
     v-toolbar-items(hidden-sm-and-down)
       v-btn(flat)
-        v-icon mdi-forum
-        span Forum
+        v-icon(left) mdi-forum
+        span(v-if="$vuetify.breakpoint.mdAndUp") Forum
       v-btn(flat href="../konnekt")
-        v-icon mdi-web
-        span Konnekt
+        v-icon(left) mdi-web
+        span(v-if="$vuetify.breakpoint.mdAndUp") Konnekt
       v-menu
         template(v-slot:activator="{ on }")
           v-btn(flat v-on="on")
-            v-icon mdi-account
-            span Account
-            v-icon mdi-menu-down
+            v-icon(left) mdi-account
+            span(v-if="$vuetify.breakpoint.mdAndUp") Account
+            v-icon(v-if="$vuetify.breakpoint.mdAndUp") mdi-menu-down
         v-list
           v-list-tile(v-for="item in AccountItems")
```

Comment:
This should also have a `:key="i"`, use `v-list-tile(v-for="(item, i) in accountItems" :key="i")`

Reply:
Also, instance variables should start with lower case `accountItems`

Reply:
Okay I will make sure that doesn't happen next time.